test/integration/smoke/test_scale_vm.py

```diff
@@ -91,6 +99,7 @@ def setUpClass(cls):
             mode=cls.services["mode"]
         )
         cls._cleanup = [
```

Comment:
can we do a "proper" cleanup construct here and not expand on this one, now that we are touching this file? see `smoke.test_accounts.TestRemoveUserFromAccount` as example.